lib/List/styles.scss

```diff
@@ -49,7 +47,18 @@ $list-border: 1px solid $color-border-base;
   }
 
   .is-current > & {
-    background: #f9f9f9;
+    background: #EBF5FD;
```

Comment:
Do we have a setting for this?

Reply:
@kyleharper we don't. I attempted to lighten up our $primary-blue colour but once lightened it gave off a purplish hue. Shall I add this to gather-ui as a $primary-light-blue colour?